src/models/api/WorkPoolResponse.ts

```diff
@@ -14,6 +14,7 @@ export type WorkPoolResponse = {
   description: string | null,
   type: string,
   is_paused: boolean | null,
+  is_push_pool: boolean | null,
```

Comment:
Nit: this doesn't change anything functionally but since this actually won't exist in oss I would change this slightly
```suggestion
  is_push_pool?: boolean,
```